lib/providers/auth0.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+exports = module.exports = function (options) {
+
+    options = options || {};
```

Comment:
You should be enforcing that `domain` exists and is a string. See phabricator.js for an example.